processing/src/main/java/org/apache/druid/segment/vector/VectorSelectorUtils.java

```diff
@@ -57,7 +57,8 @@
         final int row = i + startOffset;
         nullIterator.advanceIfNeeded(row);
         if (!nullIterator.hasNext()) {
-          break;
```

Comment:
How about using `Arrays.fill` to fill the rest with falses, and then break? Should be somewhat more efficient.

Reply:
:+1: will change